src/Relation/Nullary.agda

```diff
@@ -10,7 +10,10 @@
 
 module Relation.Nullary where
 
-open import Agda.Builtin.Equality
+open import Agda.Builtin.Equality using (_≡_)
+open import Agda.Builtin.Maybe public
+  using (nothing; just)
+  renaming (Maybe to WeaklyDecidable)
```

Comment:
Anything that we're opening publicly usually doesn't go in the import list, but under a heading in the main file to make it more obvious that some public re-exporting is going on.

Of course, per my comment above, I'm not sure we should be exporting them publicly. I would prefer a simple
```agda
WeaklyDecidable : Set a -> Set a
WeaklyDecidable = Maybe
```
and leave them to import `Data.Maybe` if they need to.

Reply:
Ah...

Reply:
So my feeling is that we should be renaming something if we're really considering the same concept, but just want a different name for it. Here it seems at least to me this is a different concept that happens to coincide at the same type... but I admit this isn't precise!

Reply:
I think that's not *quite* right (not sure, though! it's more glaring in eg #1957 / #1958 ) but in the absence of improved ux/ecosystem engineering for this kind of renaming, better not to try to solve this now!

Reply:
Another 'language desiderata' emerging right here.